### PR TITLE
init project

### PR DESCRIPTION
**Pull request by blackjmxx**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    <title></title>
```
Comment (n1c01a5) [inline index.html]: add title

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      }
+      .show-list-numbers-and-dots ul{
+        list-style-type: disc;
+        padding-left:40px;
+      }
+      .show-list-numbers-and-dots ol{
+        list-style-type: decimal;
+        padding-left:40px;
+      }
+    </style>
```
Comment (n1c01a5) [inline index.html]: add `custom.css` with this style

File: index.html (hunk 1 above)
Comment (blackjmxx) [inline index.html]: Header must be removed

File: index.html (hunk 2 above)
Comment (blackjmxx) [inline index.html]: scss must be added

**Pull request merged**